src/OptimalTransport.jl

```diff
@@ -4,6 +4,8 @@
 
 module OptimalTransport
 
+using PDMats: length
+using LinearAlgebra: AbstractMatrix
```

Comment:
Both `length` and `AbstractMatrix` are not defined in these packages but in base Julia. There should be no need to load them explicitly.
```suggestion
```